### PR TITLE
Add acceptance dashboard

### PR DESCRIPTION
**Pull request by jodiex**
Added hacker invitation page and hacker info tab in the dashboard. Resources and schedule tabs are still empty.

### CODE REVIEW

File: src/components/HackerInfo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  if (!authCtx.isAuthenticated) {
+    return null;
+  }
+
+  const email = authCtx.user.email;
+  const userConfirmed = authCtx.user.status.confirmed;
+
+  return (
+    <div className={styles.container}>
+      {!userConfirmed && <Navigate replace to='/invite' />}
```
Comment (jodiex) [inline src/components/HackerInfo/index.tsx:29]: Redirects to invitation page if user has not confirmed their attendance yet.

Is this the correct user status variable to use? I saw a lot of similar ones like `isRSVPOpen` or `accepted/declined`?

File: src/components/HackerInfo/index.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      </div>
+      <div>
+        <Typography textColor='primary-3' textType='heading3' as='h3'>
+          USEFUL LINKS
+        </Typography>
+        <div className={styles.links}>
+          <IconLink
+            logo={HopinLogo}
+            title='Hopin'
+            link='https://hopin.com/'
```
Comment (jodiex) [inline src/components/HackerInfo/index.tsx:91]: Need to update links to actual link.

File: src/components/HackerInfo/index.tsx (hunk 1 above)
Comment (fpunny) [reply src/components/HackerInfo/index.tsx:29]: Hmm it should be fine, i can look into this some more :p

File: src/components/HackerInfo/index.tsx (hunk 2 above)
Comment (fpunny) [reply src/components/HackerInfo/index.tsx:91]: sounds good, something i personally like doing for things like this is to add a comment like this
```// TODO: xxxxx```

there are extensions on VScode which find those and makes a checklist for u. for example
https://linuxpip.org/vscode-todo-list/#:~:text=TODO%20Tree%20is%20a%20VSCode,to%20be%20done%20right%20away.

**Pull request merged**